### DISCUSSION TITLE
switch to fluidd?

### DISCUSSION THREAD

**Discussion by dasb00ter** [bassamanator/Sovol-SV06-firmware#171] [Q&A] [answered]
Did I miss this how can I try out fluidd? After set up with this github. Can I screw up the mainsail set up trying the other frontend

**Comment by bassamanator** [comment, marked as answer]
These are just front-ends, hard to screw them up. Install fluidd via kiauh, but i believe only one front end can be at play, so you might have to uninstall mainsail, but kiauh does that as well.
